### ISSUE TITLE
0.5.0-rc2 qt linking error

### ISSUE THREAD

**Issue by rapus95** [JuliaPy/Conda.jl#42]
coming from there: https://github.com/stevengj/PyPlot.jl/issues/237
that's the error now:

``` julia
julia> using Conda

julia> Conda.add("pyqt")
Get-Process : Es kann kein Prozess mit dem Namen "Outlook" gefunden werden. Überprüfen Sie den Prozessnamen, und rufen Sie das Cmdlet erneut auf.
In Zeile:1 Zeichen:1
+ Get-Process Outlook
+ ~~~~~~~~~~~~~~~~~~~
    + CategoryInfo          : ObjectNotFound: (Outlook:String) [Get-Process], ProcessCommandException
    + FullyQualifiedErrorId : NoProcessFoundForGivenName,Microsoft.PowerShell.Commands.GetProcessCommand

Fetching package metadata .........
Solving package specifications: ..........

Package plan for installation in environment C:\Users\necka\.julia\v0.5\Conda\deps\usr:

The following NEW packages will be INSTALLED:

    pyqt: 4.11.4-py27_7
    qt:   4.8.7-vc9_9   [vc9]

Linking packages ...
Error: Error: post-link failed for: qt-4.8.7-vc9_9                       |   0%
ERROR: failed process: Process(setenv(`'C:\Users\necka\.julia\v0.5\Conda\deps\usr\Scripts\conda' install -y pyqt`,String["CONDARC=C:\\Users\\necka\\.julia\\v0.5\\Conda\\deps\\usr\\condarc-julia","=::=::\\"]), ProcessExited(1)) [1]
 in pipeline_error(::Base.Process) at .\process.jl:616
 in run(::Cmd) at .\process.jl:592
 in add(::String) at C:\Users\necka\.julia\v0.5\Conda\src\Conda.jl:144
```

**Comment by stevengj** [comment]
Try opening the Windows command line and running `C:\Users\necka\.julia\v0.5\Conda\deps\usr\ Scripts\conda install -y pyqt` to see if the error recurs.  If so, then the problem may be with with Anaconda itself, not Conda.

It sounds similar to ContinuumIO/anaconda-issues#555 or especially https://github.com/conda/conda#1811 ... see the comments in those issues for suggestions.

(Note: I found those issues by googling `"post-link failed" qt`.   Googling keywords from the error message is always a good first thing to try when you encounter a problem.)

![](https://imgs.xkcd.com/comics/wisdom_of_the_ancients.png)

**Comment by rapus95** [comment]
running the command you recommended indeed WORKED and solved my problem. So when it was executed from within julia it didn't work, but running it from cmd directly it worked... That's strange.
Thanks for solving.

Close the problem bcs there is a solution or keep it open because you need to workaround?

**Comment by stevengj** [comment]
I guess keep it open?  The problem with this sort of thing is reproducing it.

**Comment by Luthaf** [comment]
Could this be the same issue that #41 is trying to solve (i.e. a linking error of Qt on Windows?)

**Comment by stevengj** [comment]
@Luthaf, that sounds like a possibility.

**Comment by Luthaf** [comment]
It looks like this have been resolved with 0.5-rc3. @rapus95, could you try to check if this works for you?

**Comment by stevengj** [comment]
Seems resolved?
